fe/fe-core/src/main/java/org/apache/doris/catalog/FunctionSet.java

```diff
@@ -241,6 +242,11 @@ public Function getFunction(Function desc, Function.CompareMode mode, boolean is
                 inferenceFunctions.add(fn);
                 continue;
             }
+            // datev1 function signature can not be normal one
+            if (fn.getFunctionName().getFunction().equalsIgnoreCase("to_date")
+                    && Config.disable_datev1 && fn.getReturnType() == Type.DATE) {
+                continue;
```

Comment:
i think we should not remove this function signature. we should do cast datev1 to datev2 when we create table and insert data

Reply:
get